test-schema.json

```diff
@@ -56,6 +70,18 @@
                 }
             },
             "additionalProperties": false
+        },
+        "spec": {
+            "properties": {
+                "section": {
+                    "type": "string",
+                    "pattern": "^\\d+(\\.\\d+)*\\.$"
```

Comment:
This seems wrong (in that it ends in a period -- presumably that's simply a formatting period if it appears in the doc) but also too specific to me. Not all specifications will have this form of heading sections, it's perfectly believable that one will use e.g. letters in the section names, so `I.2.IV` or whatever.

I see @gregsdennis left you some helpful comments, I'd read those, and then probably we could even consider having different definitions of `section` depending on the allowed specification values if we really want to make sure they're correct.

Reply:
The regex I left was an example, if we wanted to even enforce such a thing.

Reply:
So, sections can also contain 1.A, 1.2.B, I.V.1.a, anything right ? so we can check for only periods in between?